DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -1486,6 +1486,79 @@ function SetWatchFlags() {
     }
 }
 
+function bool FailIfCorpseNotHeld(class<#var(DeusExPrefix)Carcass> carcClass, string goal)
+{
+    local #var(PlayerPawn) p;
+
+    p = player();
+    if (POVCorpse(p.inHand) == None || string(carcClass) != POVCorpse(p.inHand).carcClassString) {
+        class'DXREventsBase'.static.MarkBingoAsFailed(dxr, goal);
+        return true;
+    }
+    return false;
+}
+
+function MarkBingoFailedSpecial()
+{
+    local int progress, maxProgress;
+    local PlayerDataItem data;
+
+    data = class'PlayerDataItem'.static.GiveItem(player());
+
+    switch (dxr.localURL) {
+    case "02_NYC_BATTERYPARK":
+    case "03_NYC_UNATCOISLAND":
+    case "03_NYC_BATTERYPARK":
+        FailIfCorpseNotHeld(class'TerroristCommanderCarcass', "LeoToTheBar");
```

Comment:
this is why classes are better than strings lol

```
C:\Games\DX\Deus Ex HX\HXRandomizer\Classes\DXREvents.uc(1514) 
: Error, Call to 'FailIfCorpseNotHeld': type mismatch in parameter 1
```

the compiler realized that this class doesn't cast into the argument type, because it needs `#var(prefix)`, I'll fix it now